install/_lib.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
+cd "$(dirname $0)/.."
```

Comment:
Maybe a comment explaining here why this is needed and what this is intending to do?

Reply:
Done in cde39996b19350443764541682b14102b3e84086.